docs/src/content/en/docs/workflows/human-in-the-loop.mdx

```diff
@@ -1,268 +1,126 @@
 ---
 title: "Human-in-the-loop | Workflows | Mastra Docs"
```

Comment:
as a reader, how am I supposed to understand the relationship between Suspend & Resume and Human-in-the-loop? 

because Suspend & Resume comes before HITL in the left-hand menu, I would add a paragraph or callout to that page explaining the relationship between this pages and what to expect

in the HTIL page, you might want to direct them to the previous Suspend & Resume page for fundamental information that serves as a prerequisite